docs/src/main/sphinx/release/release-381.md

```diff
@@ -0,0 +1,90 @@
+# Release 381 (11 May 2022)
+
+## General
+
+## Security
+
+* Add support for automatic discovery of OpenID Connect metadata with OAuth 2.0
```

Comment:
@kokosing  here is the reworded entry .. please confirm

Reply:
👍 
CC: @lukasz-walkiewicz